libs/langchain-community/src/vectorstores/milvus.ts

```diff
@@ -335,9 +349,9 @@ export class Milvus extends VectorStore {
       collection_name: this.collectionName,
       search_params: {
         anns_field: this.vectorField,
-        topk: k.toString(),
+        topk: k,
```

Comment:
This isn't necessary anymore?

Reply:
topk is a number
![image](https://github.com/langchain-ai/langchainjs/assets/7420640/0092abdd-5f37-4164-b727-711c65400b30)

Reply:
topk is a number
![image](https://github.com/langchain-ai/langchainjs/assets/7420640/0ad09c94-809f-47ca-b748-883f7b095e1c)